### PR TITLE
Get json network dependency graph

### PR DESCRIPTION
**Pull request by deepanjanroy**
Output the clovis network dependency graph as json. 

Usage:
```shell
> lighthouse --save-artifacts
> CHROMIUM_SOURCE_PATH=$HOME/chromium/src scripts/netdep-graph.sh artifacts.log
> CHROMIUM_SOURCE_PATH=$HOME/chromium/src python scripts/netdep_graph_json.py
# JSON graph saved in dependency-graph.json
```

### CODE REVIEW

File: scripts/netdep-graph.sh [hunk 1]
```diff
@@ -1,14 +1,20 @@
 #!/bin/sh
```
Comment (paulirish) [inline scripts/netdep-graph.sh]: Can probably rename this file to `/netdep-graph_png.sh` now.

File: scripts/netdep-graph.sh [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 
 # Usage:
-#     CHROMIUM_SOURCE_PATH=$HOME/chromium/src ./netdep-graph.sh
+#     CHROMIUM_SOURCE_PATH=$HOME/chromium/src ./netdep-graph.sh ../artifacts.log
 #     open network-dependency-graph.png
 
 if [[ x"$CHROMIUM_SOURCE_PATH" == x ]]; then
```
Comment (paulirish) [inline scripts/netdep-graph.sh]: wanna match your nice `if 'CHROMIUM_SOURCE_PATH' not in os.environ:` check?

Comment (deepanjanroy) [inline scripts/netdep-graph.sh]: Do you mean changing the error message to "You must set the environment variable CHROMIUM_SOURCE_PATH"?

File: .gitignore [hunk 3]
```diff
@@ -7,4 +7,4 @@ npm-debug.log
 artifacts.log
 clovis-trace.log
 network-dependency-graph.png
-
+dependency-graph.json
```
Comment (brendankenny) [inline .gitignore]: not specific to this PR, but I'm going to want a `npm run testclean` or something pretty soon for all the artifacts our tests are accumulating without using `git clean` and losing anything *I* might have locally created on purpose :)

File: scripts/netdep-graph-png.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```
Comment (brendankenny) [inline scripts/netdep-graph-png.sh]: `git mv` next time :)

**Comment by brendankenny** [comment]
Nice! :red_circle:🕸🔵

File: scripts/netdep-graph-png.sh (hunk 4 above)
Comment (paulirish) [inline scripts/netdep-graph-png.sh]: he probably did, but 50% similarity breaks with small files pretty fast.

in this case, the usage lines were added in the middle of the file, so it no longer hit the 50% similarity metric.

**Pull request merged**